lib/src/types.dart

```diff
@@ -227,6 +289,24 @@ abstract class Type<T extends Object> {
   /// Must be a [Point]
   static const point = GenericType<Point>(TypeOid.point);
 
+  /// Must be a [Line]
+  static const line = GenericType<Line>(TypeOid.line);
+
+  /// Must be a [Lseg]
+  static const lseg = GenericType<Lseg>(TypeOid.lseg);
```

Comment:
I'm a bit undecided if this should be `lseg` or `lineSegment`. I think it may be worth to expand the name (also in `TypeOid`).

Reply:
Changed everything to lineSegment/LineSegment